// @openmeteo/sdk/probability.js
"use strict";
// automatically generated by the FlatBuffers compiler, do not modify
Object.defineProperty(exports, "__esModule", { value: true });
exports.Probability = void 0;
/* eslint-disable @typescript-eslint/no-unused-vars, @typescript-eslint/no-explicit-any, @typescript-eslint/no-non-null-assertion */
var Probability;
(function (Probability) {
    Probability[Probability["none"] = 0] = "none";
    Probability[Probability["gt0"] = 1] = "gt0";
    Probability[Probability["gt0p1"] = 2] = "gt0p1";
    Probability[Probability["gt1"] = 3] = "gt1";
    Probability[Probability["gt2"] = 4] = "gt2";
    Probability[Probability["gt3"] = 5] = "gt3";
    Probability[Probability["gt4"] = 6] = "gt4";
    Probability[Probability["gt5"] = 7] = "gt5";
    Probability[Probability["gt8"] = 8] = "gt8";
    Probability[Probability["gt10"] = 9] = "gt10";
    Probability[Probability["gt15"] = 10] = "gt15";
    Probability[Probability["gt20"] = 11] = "gt20";
    Probability[Probability["gt25"] = 12] = "gt25";
    Probability[Probability["gt30"] = 13] = "gt30";
    Probability[Probability["gt35"] = 14] = "gt35";
    Probability[Probability["gt40"] = 15] = "gt40";
    Probability[Probability["gt50"] = 16] = "gt50";
    Probability[Probability["gt60"] = 17] = "gt60";
    Probability[Probability["gt75"] = 18] = "gt75";
    Probability[Probability["gt80"] = 19] = "gt80";
    Probability[Probability["gt100"] = 20] = "gt100";
    Probability[Probability["lt0"] = 21] = "lt0";
    Probability[Probability["lt0p1"] = 22] = "lt0p1";
    Probability[Probability["lt1"] = 23] = "lt1";
    Probability[Probability["ltm1"] = 24] = "ltm1";
    Probability[Probability["ltm2"] = 25] = "ltm2";
    Probability[Probability["ltm3"] = 26] = "ltm3";
    Probability[Probability["ltm4"] = 27] = "ltm4";
    Probability[Probability["ltm5"] = 28] = "ltm5";
    Probability[Probability["ltm8"] = 29] = "ltm8";
    Probability[Probability["ltm10"] = 30] = "ltm10";
    Probability[Probability["ltm15"] = 31] = "ltm15";
    Probability[Probability["ltm20"] = 32] = "ltm20";
    Probability[Probability["gt1stdev"] = 33] = "gt1stdev";
    Probability[Probability["gt1p5stdev"] = 34] = "gt1p5stdev";
    Probability[Probability["gt2stdev"] = 35] = "gt2stdev";
    Probability[Probability["lt1stdev"] = 36] = "lt1stdev";
    Probability[Probability["lt1p5stdev"] = 37] = "lt1p5stdev";
    Probability[Probability["lt2stdev"] = 38] = "lt2stdev";
})(Probability || (exports.Probability = Probability = {}));
